### ISSUE TITLE
Request for null-safety upgrade

### ISSUE THREAD

**Issue by marvelvv** [devdennysegura/flutter-auth0#49]
I am trying to upgrade my package to sdk: ">=2.12.0 <3.0.0" which uses flutter_auth and flutter_core, but is unable to resolve the versions, is there any plan to upgrade to null-safety soon?

![Screen Shot 2021-04-27 at 11 00 30 AM](https://user-images.githubusercontent.com/47638667/116268097-fda80e80-a74a-11eb-9516-bfe215c931f9.png)

**Comment by Lampat21** [comment]
Is there any update on the null safety? This plugin is the only one that works decently covering all the auth params I need... Does anyone have any solution on the null safety migration??

**Comment by RogueBumbler** [comment]
Yes I am also desperately looking for a fix here.  Still stuck on Flutter 2.2.3!  I am sure I read somewhere a workaround that was successfully implemented?  Any help - much appreciated!
